movement/watch_faces/complication/totp_face.c

```diff
@@ -31,42 +31,61 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 // Enter your TOTP key data below
-static const uint8_t num_keys = 2;
-static uint8_t keys[] = {
-    0x48, 0x65, 0x6c, 0x6c, 0x6f, 0x21, 0xde, 0xad, 0xbe, 0xef, // 1 - JBSWY3DPEHPK3PXP
-    0x48, 0x65, 0x6c, 0x6c, 0x6f, 0x21, 0xde, 0xad, 0xbe, 0xef, // 2 - JBSWY3DPEHPK3PXP
-};
-static const uint8_t key_sizes[] = {
-    10,
-    10,
-};
-static const uint32_t timesteps[] = {
-    30,
-    30,
-};
-static const char labels[][2] = {
-    { '2', 'F' },
-    { 'A', 'C' },
+
+/* The default key size is 20, the default algorithm is SHA1 and
+ * the default timestep is 30 seconds.
+ *
+ * A label is made up of two characters (Which can be entered as a string.)
+ * Due to the structure of the display, the first character can be
+ * displayed as anything but an uppercase R.
+ * The second character can be displayed as the letters A, B, C, D, E,
+ * F, H, I, J, L, N, O, R, T, U and X, and the numbers 0, 1, 3, 7 and 8.
+ * (See: https://www.sensorwatch.net/docs/wig/display/)
+ *
+ * Ignore the initializer-overrides warning for the credentials array.
+ * It is wanted behaviour in this instance.
+ */
+#pragma GCC diagnostic ignored "-Winitializer-overrides"
+const static totp_parameters_t credentials[] = {
+  CREDENTIAL(.label = "2F", .key_size = 10),
+  CREDENTIAL(.label = "AC", .key_size = 10),
+  CREDENTIAL(.label = "GL"), // Using the default key size (20).
```

Comment:
Examples are good here, but maybe there should be a timestep example as well? I know it's not likely to be modified but it is still something you can edit from what I can tell. (I'm assuming it's as simple as `.timestep = 31` for example).